src/timepicker/timepicker.ts

```diff
@@ -63,17 +63,23 @@ const NGB_TIMEPICKER_VALUE_ACCESSOR = {
       <tr>
         <td>
           <input type="text" class="form-control" maxlength="2" size="2" 
-            [ngModel]="formatHour(model?.hour)" (ngModelChange)="updateHour($event)">
+            [ngModel]="formatHour(model?.hour)" (ngModelChange)="updateHour($event)"
+            (keyup.arrowUp)="changeHour(hourStep)" (keyup.arrowDown)="changeHour(-hourStep)"
```

Comment:
I'm not sure `keyup` is the best event as we can't prevent default on it. IMO we should use `keydown` _and_ prevent default.